cmd/hoverfly/main.go

```diff
@@ -106,6 +118,45 @@ func main() {
 		log.SetFormatter(&log.TextFormatter{})
 	}
 
+	if *generateCA {
```

Comment:
I think this main method is getting quite long, is there a way to split things up a bit more?